modules/oneVideoBidAdapter.js

```diff
@@ -0,0 +1,193 @@
+import * as utils from 'src/utils';
+import {registerBidder} from 'src/adapters/bidderFactory';
+const BIDDER_CODE = 'oneVideo';
+export const spec = {
+  code: 'oneVideo',
+  ENDPOINT: '//ads.adaptv.advertising.com/rtb/openrtb?ext_id=',
+  SYNC_ENDPOINT1: 'https://cm.g.doubleclick.net/pixel?google_nid=adaptv_dbm&google_cm&google_sc',
+  SYNC_ENDPOINT2: 'https://pr-bh.ybp.yahoo.com/sync/adaptv_ortb/{combo_uid}',
+  SYNC_ENDPOINT3: 'https://sync-tm.everesttech.net/upi/pid/m7y5t93k?redir=https%3A%2F%2Fsync.adap.tv%2Fsync%3Ftype%3Dgif%26key%3Dtubemogul%26uid%3D%24%7BUSER_ID%7D',
+  SYNC_ENDPOINT4: 'https://match.adsrvr.org/track/cmf/generic?ttd_pid=adaptv&ttd_tpi=1',
+  supportedMediaTypes: ['video'],
+  /**
+   * Determines whether or not the given bid request is valid.
+   *
+   * @param {BidRequest} bid The bid params to validate.
+   * @return boolean True if this is a valid bid, and false otherwise.
+   */
+  isBidRequestValid: function(bid) {
+    if (bid.bidder !== BIDDER_CODE || typeof bid.params === 'undefined') {
+      return false;
+    }
+
+    // Video validations
+    if (typeof bid.params.video === 'undefined' || typeof bid.params.video.playerWidth === 'undefined' || typeof bid.params.video.playerHeight == 'undefined' || typeof bid.params.video.mimes == 'undefined') {
+      return false;
+    }
+
+    // Pub Id validation
+    if (typeof bid.params.pubId === 'undefined') {
+      return false;
+    }
+
+    return true;
+  },
+  /**
+   * Make a server request from the list of BidRequests.
+   *
+   * @param {validBidRequests[]} - an array of bids
+   * @return ServerRequest Info describing the request to the server.
+   */
+  buildRequests: function(bids) {
+    return bids.map(bid => {
+      console.log(getRequestData(bid));
```

Comment:
please remove all debugging information.  there shouldn't be any `console.log` in the production code (other than through `utils.logWarn`, `utils.logError`, etc.

Reply:
done